dist/dfu.js

```diff
@@ -134,6 +134,9 @@
             })
             .then(function() {
                 log("modeData written");
+                setTimeout(function() {  // TODO: Remove this when gattserverdisconnected event is implemented and possibly put a timeout in that event handler before resolving.
```

Comment:
I'm confused about the differences between node example and web_html. Are the differences because you are testing on Android and I am testing on chrome os? It seems like the disconnect event has been working for you and you have been able to do a full DFU while I never have been able to on chrome OS with web_example.html. (and I don't see how node would have an effect on this). do you know why we are seeing differences?